### PR TITLE
Use internal norm in the stiffness detection

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
Fixes https://github.com/SciML/OrdinaryDiffEq.jl/issues/1617

### CODE REVIEW

File: src/perform_step/low_order_rk_perform_step.jl [hunk 1]
```diff
@@ -818,9 +818,9 @@ end
     # Hairer II, page 22
     ϱu, ϱd = zero(eltype(k7))^2, zero(eltype(g7))^2
     @. g6 = (g7 - g6)^2
```
Comment (devmotion) [inline src/perform_step/low_order_rk_perform_step.jl:820]: This should be changed I guess:
```suggestion
    @. g6 = g7 - g6
```

File: src/perform_step/low_order_rk_perform_step.jl [hunk 2]
```diff
@@ -818,9 +818,9 @@ end
...
-    ϱd = sum(g6)
+    ϱd = integrator.opts.internalnorm(g6,t)
     @. tmp = (k7 - k6)^2
```
Comment (devmotion) [inline src/perform_step/low_order_rk_perform_step.jl:822]:
```suggestion
    @. tmp = k7 - k6
```

File: src/perform_step/low_order_rk_perform_step.jl [hunk 3]
```diff
@@ -817,10 +817,10 @@ end
...
     ϱu, ϱd = zero(eltype(k7))^2, zero(eltype(g7))^2
-    @. g6 = (g7 - g6)^2
-    ϱd = sum(g6)
-    @. tmp = (k7 - k6)^2
-    ϱu = sum(tmp)	
+    @. g6 = g7 - g6
+    ϱd = integrator.opts.internalnorm(g6,t)
+    @. tmp = k7 - k6
+    ϱu = integrator.opts.internalnorm(tmp,t)
     integrator.eigen_est = sqrt(ϱu/ϱd)*oneunit(t)
```
Comment (danielwe) [inline src/perform_step/low_order_rk_perform_step.jl:824]: I think you need to drop the sqrt here

Comment (danielwe) [reply src/perform_step/low_order_rk_perform_step.jl]: Also noticing that `oneunit(t)` is only used here and not in similar code for other algorithms. Any reason for that?

**Pull request merged**